typescript/infra/config/environments/testnet4/igp.ts

```diff
@@ -22,8 +22,6 @@ export const storageGasOracleConfig: AllStorageGasOracleConfigs =
     gasPrices,
     (local, remote) =>
       getTokenExchangeRateFromValues(local, remote, tokenPrices),
-    (local) => parseFloat(tokenPrices[local]),
```

Comment:
introduced in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4562 but we don't really need to provide these functions on testnet